sda-commons-shared-asyncapi/src/main/java/org/sdase/commons/shared/asyncapi/schema/JsonSchemaExamplesModule.java

```diff
@@ -0,0 +1,59 @@
+package org.sdase.commons.shared.asyncapi.schema;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.github.victools.jsonschema.generator.FieldScope;
+import com.github.victools.jsonschema.generator.Module;
+import com.github.victools.jsonschema.generator.SchemaGenerationContext;
+import com.github.victools.jsonschema.generator.SchemaGeneratorConfigBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+// TODO: Should we contribute this? This would be a good default behavior for victools.
```

Comment:
This file is something that we should contribute to victools. Having such a annotation there would be quite useful.